cli/denort.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+mod flags_rt;
+mod rt;
+mod version;
+
+fn main() {
+  #[cfg(windows)]
+  deno_runtime::colors::enable_ansi(); // For Windows 10
+
+  let args: Vec<String> = std::env::args().collect();
+  if let Err(err) = rt::try_run_standalone_binary(args) {
+    eprintln!(
+      "{}: {}",
+      deno_runtime::colors::red_bold("error"),
+      err.to_string()
+    );
+    std::process::exit(1);
+  }
+
+  // TODO (yos1p) Specify better error message
+  eprintln!(
+    "{}: Runtime Error.",
```

Comment:
@ry Any suggestions for the error message?

Reply:
@yos1p this is fine for now, I will review this PR in more detail today/tomorrow and merge it. @lucacasonato and I will look into using this new binary target for standalone executables.

Reply:
"This executable is used internally by 'deno compile', it is not meant to be invoked directly."